tests/utils/string_test.py

```diff
@@ -0,0 +1,60 @@
+import unittest
+
+from aicord.utils.string import split_paragraph_chunks
+
+
+class TestSplitParagraphChunks(unittest.TestCase):
+    def test_short_text(self):
+        """
+        Test case where the text is shorter than the maximum length.
+        """
+        text = "This is a short paragraph."
+        max_length = 100
+        expected_output = ["This is a short paragraph."]
+        self.assertEqual(split_paragraph_chunks(text, max_length), expected_output)
+
+    def test_exact_length_text(self):
+        """
+        Test case where the text is exactly the maximum length.
+        """
+        text = "a" * 100
```

Comment:
Remove all weird test data like this. All test data have to be human-readable and understandable